src/Factories/FunFunFactory.php

```diff
@@ -684,6 +686,15 @@ function () {
 			} );
 	}
 
+	private function getSalutations(): Salutations {
+		return $this->createSharedObject( Salutations::class,
+			function () {
+				$json = ( new SimpleFileFetcher() )->fetchFile( $this->getI18nDirectory() . '/data/salutations.json' );
+				$data = json_decode( $json, true, 16, JSON_THROW_ON_ERROR );
+				return new Salutations( $data[ 'salutations' ] );
```

Comment:
The data in `salutations.json` could also be an array, without the need for a wrapper object. Or did you have any future extensions in mind that would benefit form an object?

Reply:
I was trying to get a little type safety